Source/Core/Xml.cs

```diff
@@ -121,6 +121,14 @@ public void WriteStartSplit(int splitNum, DateTime startTime)
         wr.WriteStartElement("split");
         wr.WriteAttributeString("number", splitNum.ToString());
         wr.WriteAttributeString("startTime", startTime.ToString(DateTimeFormatString));
+
+        if (token != null) {
+          wr.WriteStartElement("location");
```

Comment:
I'm a bit worried about attaching a single "location" to splits: it's obviously useful if you happened to use `/vcsSplitOnEveryAssert`, but it could be very confusing if you're using `{split_here}` and/or `{:focus}` and happen to end up with some splits with only one assert in them. It would be more general to log the location of EVERY assertion in a split IMO, but that would increase the XML size substantially if it was on all the time.

Let's discuss the options a bit more before merging this.

Reply:
Good call. I was thinking primarily of the `/vcsSplitOnEveryAssert` case, and this could perhaps be a little strange in other cases.

Reply:
Just for experimentation, I pushed a commit that includes locations for every assert within a split, which seems, on further consideration, like the most informative thing for a user. It _doesn't_ list anything when there are no splits (since it wouldn't add any information; you'll know the proof includes all assertions at once).